tests/unit/deployment/test_dashboard.py

```diff
@@ -0,0 +1,138 @@
+from pathlib import Path
+from unittest.mock import create_autospec
+
+from databricks.labs.blueprint.installation import MockInstallation
+from databricks.sdk import WorkspaceClient
+from databricks.sdk.errors import InvalidParameterValue
+from databricks.sdk.service.dashboards import Dashboard
+
+from databricks.labs.remorph.contexts.application import CliContext
+from databricks.labs.remorph.deployment.dashboard import DashboardDeployment
+
+
+def test_deploy_new_dashboard():
```

Comment:
don't cover this with tests - it has to be replaced with https://github.com/databrickslabs/ucx/pull/1920#issuecomment-2218048976

Reply:
Dashboard deployment using lsql will be implemented as a separate issue